test/test_integration_ssl.rb

```diff
@@ -33,7 +33,6 @@ def with_server(config)
     config_file = Tempfile.new %w(config .rb)
     config_file.write config
     config_file.close
-    config_file.path
```

Comment:
👍 

[Looks like a left-over from when the code looked different](https://github.com/puma/puma/commit/09be7ce9234904944d827a32ce1a2bcb42a1e763#diff-965243a94ff85e7b5cade7d2883887e43482c40c1104bb7453928c9b1aa705a7R60) (was moved along in https://github.com/puma/puma/commit/5608248c13130740ca94697b63a59245140e8092)